pkg/cluster/config/register.go

```diff
@@ -21,18 +21,21 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-// GroupName is the group name used in this package
+// GroupName is the group name used in this package.
 const GroupName = "kind.sigs.k8s.io"
 
 var (
-	// SchemeGroupVersion is group version used to register these objects
+	// SchemeGroupVersion is group version used to register these objects.
 	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+	// SchemeBuilder is a type to collect functions that add things to a schema.
 	SchemeBuilder      runtime.SchemeBuilder
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
+	// AddToScheme applies all the sotred functions in the localSchemeBuilder
```

Comment:
`s/sotred/sorted`

Reply:
haha it was actually supposed to be "stored."